packages/create-flex-plugin/templates/js/src/DemoPlugin.js

```diff
@@ -13,15 +16,31 @@ export default class {{pluginClassName}} extends FlexPlugin {
    * This code is run when your plugin is being started
    * Use this to modify any UI components or attach to the actions framework
    *
-   * @param flex { typeof import('@twilio/flex-ui') }
-   * @param manager { import('@twilio/flex-ui').Manager }
+   * @param flex { typeof Flex }
+   * @param manager { Flex.Manager }
    */
   init(flex, manager) {
-    flex.AgentDesktopView.Panel1.Content.add(
-      <CustomTaskListComponent key="demo-component" />,
-      {
-        sortOrder: -1,
-      }
-    );
+    this.registerReducers(manager);
+
+    const options = { sortOrder: -1 };
+    flex.AgentDesktopView
+      .Panel1
+      .Content
+      .add(<CustomTaskListContainer key="demo-component" />, options);
+  }
+
+  /**
+   * Registers the plugin reducers
+   *
+   * @param manager { Flex.Manager }
+   */
+  registerReducers(manager) {
+    if (!manager.store.addReducer) {
+      // eslint: disable-next-line
+      console.error(`You need FlexUI > 1.9.0 to use built-in redux; you are currently on ${VERSION}`);
```

Comment:
You can easily avoid disabling eslint with:
```
console.error(
  'You need FlexUI > 1.9.0 to use built-in redux;',
  `you are currently on ${VERSION}`
);
```

Reply:
The disable is for the use of `console` not length of the line

Reply:
In that case maybe only `console.log` should trigger a linting error
`no-console: ["error", { allow: ["warn", "error"] }]`

Reply:
wow, mind is blown!